plugins/cosmos-crypto/Makefile

```diff
@@ -0,0 +1,82 @@
+.PHONY: clean
+clean: clean-build clean-pyc clean-test clean-docs
+
```

Comment:
why its own Makefile?

Reply:
because I consider each plugin as an isolated repository

Reply:
Hm the Makefile is a bit of an overkill imo. It's not its own repository, just its own module/library.

Reply:
removed